companies.html

```diff
@@ -488,6 +488,17 @@
       Uses Scrapy to crawl and scrape taxi and transportation sites to gather rates, coverage zones, and deals.</p>
     </div>
 
+    <div class="company-box">
+      <a href="https://www.allogarage.fr/">
+      <img src="../img/50-allogarage-logo.png" />
```

Comment:
Impossible to add the file
here it is:
http://www.allogarage.fr/images/50-allogarage-logo.png